test/error.jl

```diff
@@ -139,3 +139,24 @@ end
 using .Issue54029
 @test Base.infer_return_type(raise54029, (Any,)) === Union{}
 @test Base.infer_return_type(xs->raise54029(xs...), (Vector{Any},)) === Union{}
+
+@testset "CompositeException" begin
+    ce = CompositeException()
+    @test isempty(ce)
+    @test length(ce) == 0
+    @test eltype(ce) == Any
```

Comment:
I don't think we should have a method for this (#54177). It doesn't hurt to test it, though it would be plausible for this eltype to turn into an abstract exception type in the future